templates/openebs-operator-1.9.0.yaml

```diff
@@ -909,13 +664,9 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
-        - name: OPENEBS_SERVICEACCOUNT_NAME
-          valueFrom:
-            fieldRef:
-              fieldPath: spec.serviceAccountName
         - name: OPENEBS_IO_CSTOR_TARGET_IMAGE
           value: "quay.io/openebs/cstor-istgt:1.8.0"
         - name:  OPENEBS_IO_CSTOR_VOLUME_MGMT_IMAGE
-          value: "quay.io/openebs/cstor-volume-manager:1.8.0"
+          value: "quay.io/openebs/cstor-volume-mgmt:1.8.0"
```

Comment:
Is this change for 1.8 or only starting with 1.9?

Reply:
Only from 1.9.0 onwards Vishnu

Reply:
I mean this is the YAML which is present in OpenEBS/OpenEBS repo, we made use of yaml manifests present in cstor-operators repo, so we had to make this change.